templates/_layouts/examples.html

```diff
@@ -27,11 +27,16 @@
     {% endif %}
 
     <script defer src="{{ versioned_static('js/example-tools.js') }}"></script>
+    {% else %}
+      <meta name="combined_template_name" content="{{ self._TemplateReference__context.name }}">
     {% endif %}
   
   </head>
 
   <body {% if is_dark %}class="is-dark"{% elif is_paper %}class="is-paper"{% endif %}>
+    {% if is_combined %}
+    <small>{{ self._TemplateReference__context.name }}</small>
```

Comment:
That's very much needed indeed. It would be annoying not to know which example to look at.

Could we make this into a link? So that when visiting combined example in the browser we could click to get to single one directly?

Reply:
@bartaz 
Changed it to a link:
![image](https://github.com/canonical/vanilla-framework/assets/46915153/00744b98-8c95-4141-9657-b2ae744b8838)